cl/api/utils.py

```diff
@@ -244,7 +244,9 @@ def _get_response_ms(self) -> int:
     def _log_request(self, request):
         d = date.today().isoformat()
         user = request.user
-        client_ip, is_routable = get_client_ip(request)
+        client_ip = get_header(request, "CloudFront-Viewer-Address").split(
+            ":"
+        )[0]
```

Comment:
Why not use: `strip_port_to_make_ip_key`? 

```suggestion
        client_ip = strip_port_to_make_ip_key(request)
```